src/fe-text/statusbar-config.c

```diff
@@ -273,6 +274,25 @@ static const char *sbar_get_visibility(STATUSBAR_CONFIG_REC *rec)
 		rec->visible == STATUSBAR_VISIBLE_INACTIVE ? "inactive" : "??";
 }
 
+static CONFIG_NODE *sbar_node(const char *name, gboolean create)
+{
+        CONFIG_NODE *node;
+	STATUSBAR_CONFIG_REC *rec = statusbar_config_find(active_statusbar_group, name);
+	if (rec != NULL) {
+		name = rec->name;
+	} else if (!create) {
+		return NULL;
+	}
+
+        /* lookup/create the statusbar node */
```

Comment:
space indents

Reply:
.. I thought I had ran the almight git-clang-format  ..